MeasureAuthoringTool/mat-drupal
# Create CTA with Image block

## Description
Enabled a few modules: paragraph, twig_field_value, media
Created image styles for the CTA with image block type
Themed CTA with Image organism

## To Test
1. composer install
2. drush import config
3. compile pattern lab
4. clear cache
5. Edit the "MAT Overview" block on the homepage, providing values for the fields.
6. View the homepage, see that the block is styled appropriately. Should look like the attached screenshot:

<img width="1373" alt="Screen Shot 2020-02-26 at 9 10 53 AM" src="https://user-images.githubusercontent.com/1919263/75358089-f13ed200-5877-11ea-8db1-fec6420cb159.png">
